app/views/bookmarks/_external_work_fields.html.erb

```diff
@@ -3,23 +3,10 @@
   <dl>
     <dt class="required"><%= ew.label :url, ts('URL') + '*' %></dt>
     <dd class="required" title="URL">
-      <% if !params[:url_from_external].blank? %>
-        <%= ew.text_field :url, autocomplete_options('external_work', :autocomplete_token_limit => 1, :value => params[:url_from_external]) %>
-      <% else %>
+      <% if params[:url_from_external].blank? %>		 
         <%= ew.text_field :url, autocomplete_options('external_work', :autocomplete_token_limit => 1) %>
```

Comment:
If we're removing the autocomplete, we should also remove all of the `autocomplete_options` bits from this file -- they're used to set HTML attributes the autocomplete JavaScript relies on